File: java/secao7/Exercicio06.java
package br.com.geekuniversity.secao7;

import java.util.Scanner;

public class Exercicio06 {

	public static void main(String[] args) {
		// variáveis
		int numero;
		Scanner teclado = new Scanner(System.in);
		
		//entradas
		System.out.println("Informe um número entre 1 e 10:");
		numero = teclado.nextInt();
		
		//processamento
		while(numero < 1 || numero >10) {
			System.out.println("Informe um número entre 1 e 10:");
			numero = teclado.nextInt();
		}		
		for(int i=1; i<=10; i++) {
			System.out.printf("%d x %d = %d\n", numero, i, (numero * i));
			
		}
		
		teclado.close();
	}

}

File: python/python_secao7/Exercicio06.py
#entradas
numero = int (input("Informe um número entre 1 e 10:"))

#processamento
while (numero < 1 or numero > 10):
    numero = int (input("Informe um número entre 1 e 10:"))
    
print("Tabuada do {0}:".format(numero))
for i in range(1,11):
    print("{0} X {1} = {2}". format(numero, i, numero * i ))